src/test/test_proto_server.c

```diff
@@ -65,11 +65,23 @@ test_run(d_rank_t my_rank)
 			  "crt_group_config_save() failed. rc: %d\n", rc);
 	}
 
-	rc = crt_proto_register(&my_proto_fmt_0);
-	D_ASSERT(rc == 0);
-	rc = crt_proto_register(&my_proto_fmt_1);
-	D_ASSERT(rc == 0);
-
+	switch (test.tg_num_proto) {
+		case 4:
+			rc = crt_proto_register(&my_proto_fmt_3);
+			D_ASSERT(rc == 0);
+		case 3:
+			rc = crt_proto_register(&my_proto_fmt_2);
+			D_ASSERT(rc == 0);
+		case 2:
+			rc = crt_proto_register(&my_proto_fmt_1);
+			D_ASSERT(rc == 0);
+		case 1:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment